### PR TITLE
Override ds-branding defaults

### PR DESCRIPTION
**Pull request by xjensen**
We have another PR on the design-system repo, cagov/design-system#124, to modify some of the ds-branding component's default values. This PR aims to provide overrides for headless.cannabis.ca.gov, to ensure this site can continue to look correct alongside changes to official ds-branding.

We should wait to merge this until the ds-branding change is merged into design-system.

Let me know if any questions or concerns!

### CODE REVIEW

**Review by aaronhans — APPROVED** [review]
cool

**Comment by kkoryaka** [comment]
@xjensen Maybe there is a way to do it without overriders? Why don't we fix what we need directly in the design system?

**Comment by aaronhans** [comment]
We have color css variables, we could expand the variable set to include some spacing css vars, also some font ones so that overrides could be more organized

I think this update where we get to use the branding component on a new site is a great first step though

**Comment by xjensen** [comment]
I agree with @kkoryaka's general idea here.

We would probably need 10+ new CSS vars to implement Cannabis/Drought within the design-system as-is. We might need to rewrite the component a bit instead. I'm thinking about that now.

For the future, I'm wondering what lives in design-system vs. on each site. Juggling three repos for this change was already a challenge, and it's only going to get worse as we take on more sites.

**Comment by xjensen** [comment]
I've rewritten parts of the upstream ds-branding component. Those changes should help accommodate Drought and Cannabis without overriding CSS. It's a big change to the internal CSS; I still need to add additional documentation.

I've also added a commit here to demonstrate the changes that would be needed in Cannabis. The markup needs a few small edits. There are also a lot of new CSS variables to control font settings. No more CSS overrides. 

Please take a look! Let me know what you think. Hopefully this is a more supportable solution.

(One note: I did update the other CSS color vars here too, to match recent changes, just so I could get a good look at everything in testing. But the only real additions needed for this PR would be the font-related changes at the bottom of the cannabis.scss file.)

File: src/css/sass/colorschemes/cannabis.scss [hunk 1]
```diff
@@ -7,27 +7,30 @@
...
+  --secondary-lightest-color: #C1C9C0;
+  --secondary-hover-color: #21351F;
+
+  --highlight-color: #C0633B;
+  --highlight-dark-color: #864529;
+  --highlight-light-color: #D39276;
+  --highlight-lightest-color: #ECD0C4;
+  --secondary-hover-color: #864529;
+  
+  --org-name-state-font: "Arial", "Helvetica", sans-serif;
```
Comment (aaronhans) [inline src/css/sass/colorschemes/cannabis.scss:28]: nice variable naming
these all make sense

File: src/css/sass/colorschemes/cannabis.scss [hunk 2]
```diff
@@ -7,27 +7,30 @@
...
-  // --secondary-active-color: #4A1106;
-  // --secondary-focused-color: #4A1106;
-  // --secondary-disabled-color: #4A1106;
-  --highlight-color: #fdb81e;
-  --highlight-color-300: #FFF2D5;
 
-  --standout-color: #2f4c2c;
-  --standout-hover-color: #21351f; 
-  --logo-font: 'Lexend';
+  --secondary-color: #2F4C2C;
```
Comment (aaronhans) [inline src/css/sass/colorschemes/cannabis.scss:16]: are these color changes intentional or are they coming through from another update?

**Comment by aaronhans** [comment]
Impressive you've gotten rid of the overrides! 

Please verify color change variable values are desired

File: src/css/sass/colorschemes/cannabis.scss (hunk 2 above)
Comment (xjensen) [reply src/css/sass/colorschemes/cannabis.scss:16]: Yep, I copied over the color changes from something else that Konstantin was doing for Cannabis, just to see correct colors in dev testing. I'll defer to Konstantin's changes when I handle the merge conflict, but looks like it's all the same.

Comment (xjensen) [reply src/css/sass/colorschemes/cannabis.scss:16]: I did change the name of the --logo-font variable to match all the other variables. I didn't see any other uses of this in Cannabis, but we should probably check with @kkoryaka just to be sure.

Comment (xjensen) [reply src/css/sass/colorschemes/cannabis.scss]: Resolved conflicts with main branch.

**Comment by xjensen** [comment]
Regarding color changes, Konstantin merged the official changes into `main` a day or two ago. I just merged his changes into this branch to ensure we're all synced up.

**Pull request merged**